.github/workflows/milestone-recorder.yml

```
@@ -0,0 +1,241 @@
+            *)
+              EMOJI="📝"
+              TITLE="実装更新"
+              ;;
+          esac
+
+          # Prepare milestone entry
+          MILESTONE_ENTRY=$(cat <<EOF
+## $EMOJI $CURRENT_DATE - $TITLE
+
+### 実装内容
+$DESCRIPTION
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 149: could not find expected ':'

(syntax-check)